packages/w3up-client/src/client.js

```diff
@@ -54,6 +55,31 @@ export class Client extends Base {
     this.coupon = new CouponAPI(agentData, options)
   }
 
+  /**
+   * Verify and return Abilities.
+   *
+   * Given a list of strings representing capability names (Abilities),
+   * verify that all the strings are valid Abilities and return Abilities[].
+   *
+   * Abilities[] is still just a list of strings, but this helps us play
+   * nice with Typescript.
+   *
+   * @param {string[]} abilities
+   * @returns {import('@web3-storage/access').Abilities[]}
+   */
+  static abilities(abilities) {
+    for (const ability of abilities) {
+      if (
+        !abilitiesAsStrings.includes(
+          /** @type {import('@web3-storage/access').Abilities} */ (ability)
```

Comment:
Side note: I'm finding it super confusing that the type of a single ability is called "Abilities"...

Reply:
Yea hard agree - I've expanded the scope of this PR a bit to `s/Abilities/Ability` in the relevant places. Keeping the old names around for now but marking them deprecated to give folks a chance to update.